packages/kit/src/runtime/client/router.js

```diff
@@ -83,7 +83,11 @@ export class Router {
 		const trigger_prefetch = (event) => {
 			const a = find_anchor(/** @type {Node} */ (event.target));
 			if (a && a.href && a.hasAttribute('sveltekit:prefetch')) {
-				this.prefetch(new URL(/** @type {string} */ (a.href)));
+        const svg = typeof a.href === 'object' && a.href.constructor.name === 'SVGAnimatedString';
```

Comment:
Maybe you should refactor out a method like:
```
function get_href(a) {
    const is_svg = typeof a.href === 'object' && a.href.constructor.name === 'SVGAnimatedString';
    const href = String(is_svg ? /** @type {SVGAElement} */ (a).href.baseVal : a.href);
    const base = is_svg ? document.baseURI : undefined;
 	return new URL(/** @type {string} */ (href), base);
}
```

these lines look to basically appear below as well. then you could share them